.github/workflows/build_and_test.yml

```diff
@@ -200,18 +200,23 @@ jobs:
       run: |
         cd pytket
         pip install -e . -v
+    - name: Run doctests
+      run: |
+        cd pytket
+        pip install pytest
+        pytest --doctest-modules pytket
```

Comment:
Can these doctests be run locally using this command?

`pytest --doctest-modules pytket`

Reply:
Yes, from the `pytket` directory that works.